docs/developers/developing.rst

```diff
@@ -144,18 +144,20 @@ Code Style
 JavaScript
 ##########
 
-Hypothesis uses ESLint to help maintain style consistency. You can check your
-changes for conformance using:
+Hypothesis uses ESLint_ (a linter) and Prettier_ (an automated code formatter)
+to ensure style consistency and help prevent common mistakes. Plugins are
+available for most editors for these tools. We recommend that you set these up
+before making changes to the code.
+
+To auto-format code and run lint checks locally using the CLI, run:
```

Comment:
It might also be worth mentioning there's a plugin for editors here?

Reply:
I did mention that "plugins are available for most editors for these tools" in the paragraph above. Did you have something additional in mind?

Reply:
Oh wow. Not sure how I missed that. Durp. Never mind!